sound/soc/sof/intel/hda-dsp.c

```diff
@@ -446,6 +464,9 @@ static int hda_dsp_set_D0_state(struct snd_sof_dev *sdev,
 		    !hda_enable_trace_D0I3_S0 ||
 		    sdev->system_suspend_target != SOF_SUSPEND_NONE)
 			flags = HDA_PM_NO_DMA_TRACE;
+
+		if (hda_dsp_d0i3_playback_active(sdev))
+			flags |= HDA_PM_PG_STREAMING;
```

Comment:
I am curious does the flags has the same meaning between IPC3 and IPC4?

Reply:
I checked that this bit in ipc3 is the same as ipc4 and it is not used by IPC3 & IPC4 SOF FW now, so FW needs to support it in future

Reply:
Even in future IPC3 has different usage compared to IPC4, IPC3 implementation is separated with IPC4 one, so they don't affect each other

Reply:
In my opinion, the original definition was a copy of ipc4 definition since it is too lucky to has the same meaning.